File: 03-Problem-If/03_Problem.py
# create a function with an if statement that returns true if an integer is even or false if it is odd
# print examples with the result for both outcomes to the log


def isEven(num):
    if num % 2 == 0:
        return True
    return False

print(46, isEven(46), 93, isEven(93))

File: 03-Problem-If/03-Problem.swift
// create a function with an if statement that returns true if an integer is even or false if it is odd
// print examples with the result for both outcomes to the log


func isEven(num: Int) -> Bool {
    if (num % 2 == 0) {
        return true;
    }

    return false;
}

print(46);
print(isEven(num: 46));
print(93);
print(isEven(num: 93));
